Maple2.Server.Game/Util/XmlParseUtil.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Xml;
+
+namespace Maple2.Server.Game.Util;
+
+public static class XmlParseUtil {
+    public static Dictionary<string, string> GetParameters(string? xmlString) {
+        Dictionary<string, string> parameters = new Dictionary<string, string>();
+        XmlDocument xmlParameter = new();
```

Comment:
I think you can just do `XmlDocument.LoadXml(...)` which is cleaner.

Reply:
If I don't initialize XmlDocument beforehand, it returns `Cannot access non-static method 'LoadXml' in static context`.